### PR TITLE
Add freedom first network

### PR DESCRIPTION
**Pull request by lanmower**
add freedom first mainnet to list

### CODE REVIEW

**Comment by DenisCarriere** [comment]
- These FF endpoints don't work
- MEETONE was removed

**Comment by lanmower** [comment]
Hi dennis, our FF endpoints are operational, can you explain what you mean
with don't work?

Would you like an account?

[image: image.png]

**Comment by lanmower** [comment]
Can you please explain what you mean with don't work, and why this issue is closed?

image

Our nodes most certainly do work, would you like an account?

File: networks.json [hunk 1]
```diff
@@ -106,20 +106,34 @@
     ]
   },
   {
-    "name": "MEETONE Mainnet",
```
Comment (DenisCarriere) [inline networks.json:109]: MEETONE was removed

File: networks.json [hunk 2]
```diff
@@ -106,20 +106,34 @@
...
-    "network": "eos",
+    "prefix":"FF",
+    "network": "ff",
     "endpoints": [
       {
-        "name": "MEETONE",
+        "name": "dconnect.live",
+        "protocol": "https",
+        "port": 443,
+        "url": "dconnect.live",
```
Comment (DenisCarriere) [inline networks.json:121]: Redirects to https://www.dconnect.live/

File: networks.json [hunk 3]
```diff
@@ -106,20 +106,34 @@
...
-        "description": "MEETONE Mainnet API"
+        "url": "steem.host",
+        "description": "steem.host public node"
+      },
+      {
+        "name": "gostomp.co.za",
+        "protocol": "https",
+        "port": 443,
+        "url": "gostomp.co.za",
+        "description": "gostomp.co.za public node"
```
Comment (DenisCarriere) [inline networks.json:136]:
```
Error: connect ECONNREFUSED 127.0.0.1:8080
    at Object._errnoException (util.js:1022:11)
    at _exceptionWithHostPort (util.js:1044:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1198:14)
```

File: networks.json [hunk 4]
```diff
@@ -106,20 +106,34 @@
     ]
   },
   {
-    "name": "MEETONE Mainnet",
-    "description": "MEETONE Mainnet",
-    "owner": "MEETONE",
-    "chainId": "cfe6486a83bad4962f232d48003b1824ab5665c36778141034d75e57b956e422",
+    "name": "Freedom First Mainnet",
```
Comment (DenisCarriere) [inline networks.json:109]: Should be called `Freedom First Testnet`

Seems like there's no identifiable block producers

```
cleos -u https://dconnect.live system listproducers

Producer      Producer key                                              Url                                                         Scaled votes
evo           EOS5ap3iQbJY89hGi75tJTa2fh6iv3J2wTk5UkeHyTrJRTucBKy9e                                                                 0.1135
atechshaman   EOS8iK6Jq73YFrfWWebkhbfj4f3GdVokRHJAwcxJSLwz5sQb8BZba                                                                 0.1135
phil          EOS7UnjhXYfUnTTrpcFP9GhMF7ubcd2sSJqBNSCBii9otguUK8zTJ                                                                 0.1105
webbed        EOS7BECyXpgjXeLaFH6LrsKmZMQApy8TDJPC7B3tU6eHfgor8z69q                                                                 0.0927
freedomfirst  EOS65eQ8DtWwgyBXeYvmpjPrHo49ohrWuQjcAcWQ4tKv13AyEXiby                                                                 0.0783
youroneangel  EOS549KADkBUMJceUUyaW6wQzZ84eEgf4Nr4iwdL4d2Xda7FDKL8s     www.google.com                                              0.0767
occultcorner  EOS77UdptM6zoiGtMGRbzBnPMZ81AcMq1qozb7F3Ycv85eG6tEfcZ                                                                 0.0767
koh           EOS5K7EkWPhgETBkPtJ3KFB1P1HBb2yseGiYCQQzmFcT9Y8vQRURD                                                                 0.0751
cheeseburger  EOS5RejDrUr7hyo2hHEtBPBScfirV1bs8YrAqXe3riCxsCjNvW4d1                                                                 0.0749
moonshine     EOS84tXCUNRVAjFBZCXFjfopKgNpk4jNNZnxwqCxssVbCC11NTZrq                                                                 0.0746
supermeatboy  EOS6AjN55SaQwtLZUmTN7NU6Gp98WhRMwLNaFkxcchQ87mF1Bio6U                                                                 0.0736
zeshanyousaf  EOS8gYqrxJgr4C74JZRchZ1byfkQYPiNoy1zbYPJp2TGWg6Apc5An                                                                 0.0399
```

**Comment by lanmower** [comment]
This is the Freedom First mainnet.

The block producers are not required to expose any personal details on FF

you can visit https://dconnect.live for more informtaion.

**Comment by lanmower** [comment]
Please let me know if there's any more information you require.

FreedomFirst is an informal mainnet run by people like you and me.

It is not a giant corporate conglomerate like other mainnets.

Block Producers are encouraged, but not at all required, to list anything specific.

Please feel free to join the discussion on our discord:
https://discord.gg/qG2e8mg

or visit any of our available public gateways (which is actually hosted using our in house decentralized web built on blockchain technology)
www.dconnect.live
www.steem.host
www.surfinghtheblockchain.com

Your dismissive behavior towards our network is questionable, if there is anything specific you'd like to know we are alive, online and available to answer questions.

**Comment by theoccultcorner** [comment]
Hello, i am occultcorner. i am a block producer for the FreedomFirst Network. i will be happy to answer any questions you might have. thank you for your curiosity. Enjoy the rest of your day.

**Comment by chivesz** [comment]
Afternoon everyone. I'm a BP on FreedomFirst as well. If you need anything please feel free to reach out. Cheers!

**Pull request closed without merging**